libc/test/UnitTest/FPMatcher.h

```diff
@@ -60,10 +60,95 @@ template <typename T, TestCond Condition> class FPMatcher : public Matcher<T> {
   }
 };
 
+template <typename T, TestCond Condition> class CFPMatcher : public Matcher<T> {
```

Comment:
I tried to incorporate the changes for complex floating types in FPMatcher class but I ran into some templating issues and so created a separate class for it. Let me know if there is an alternative to this.

Reply:
a separate template is fine, a bit duplicated but making the each class clearer.